proseco/guide.py

```diff
@@ -261,53 +266,84 @@ def get_initial_guide_candidates(self):
         stars = self.stars
         dark = self.dark
 
-        # Use the primary selection filter from acq, but allow bad color
-        # and limit to brighter stars
-        ok = ((stars['CLASS'] == 0) &
```

Comment:
A corollary to the recommendation of using `ok` for boolean masks is that this should only be done for code that is immediately adjacent.  So move this statement down to [where it is used](https://github.com/sot/proseco/blob/1c14d0e1926aa105aa243b72ec3606d9bb7794ea/proseco/guide.py#L289).